docs/manifest.json

```diff
@@ -180,1149 +180,1149 @@
 		"parent": "developers"
 	},
 	{
-		"title": "Components",
-		"slug": "components",
-		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/packages/components/README.md",
-		"parent": "developers"
-	},
-	{
-		"title": "Theming for the Block Editor",
-		"slug": "themes",
-		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/designers-developers/developers/themes/README.md",
-		"parent": "developers"
-	},
-	{
-		"title": "Theme Support",
-		"slug": "theme-support",
-		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/designers-developers/developers/themes/theme-support.md",
-		"parent": "themes"
+		"title": "@wordpress/a11y",
```

Comment:
The legacy `manifest.json` should not be changing at this point, so the legacy handbook isn't updated. Updated manifests should still be targeted at `manifest-devhub.json` until we can disable the auto-updates at the legacy handbook.

Reply:
Note that the changes merely reorder the sections to fix the previous/next links, do not alter anything else (titles, slug, etc). This can be safely merged, then the DevHub manifest PR can be rebased and updated to reflect these changes as well.